CHANGES.md

```diff
@@ -19,6 +19,9 @@ Keyword
     END
 ```
 
+### Other
+- Added ``--ignore-gitignore`` flag to ignore ``.gitignore`` files and parse files listed there ([#299](https://github.com/MarketSquare/robotframework-tidy/issues/299)). 
```

Comment:
Hmm, I don't like this name, I must say. What about `--scan-all` or maybe `--parse-all-files`. I don't know... hard case 😆 On the other hand, current name is meaningful, but a bit weird :D

Reply:
Yeah I'm also not a huge fan of the name. ``scan-all`` or ``--parse-all-files`` do miss the main point behind this flag - to ignore gitignores :P . There is also ``--skip-gitignore`` as an option

Reply:
I like `--skip-gitignore` :)